### PR TITLE
0.5 compat for symbol=>Symbol

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

**Comment by stevengj** [comment]
Thanks!  Can you also update the Compat version in REQUIRE?

**Comment by quinnj** [comment]
Like so?

File: src/DecFP.jl [hunk 1]
```diff
@@ -36,7 +36,7 @@ Base.get_rounding{T<:DecimalFloatingPoint}(::Type{T}) = rounding_c2j[unsafe_load
 Base.set_rounding{T<:DecimalFloatingPoint}(::Type{T}, r::RoundingMode) = unsafe_store!(rounding, rounding_j2c[r])
 
 for w in (32,64,128)
-    BID = symbol(string("Dec",w))
+    BID = @compat(Symbol)(string("Dec",w))
```
Comment (stevengj) [inline src/DecFP.jl]: You don't need `@compat(Symbol)`, I think.  You can just write `Symbol`.   The `@compat` is only needed for 0.3, which is not supported by DecFP.

**Comment by stevengj** [comment]
I think this PR is obsolete?

**Pull request closed without merging**